src/distilabel/steps/argilla/text_generation.py

```diff
@@ -151,13 +159,13 @@ def process(self, inputs: StepInput) -> "StepOutput":  # type: ignore
                 generations_set.add(generation)
 
                 records.append(
-                    rg.FeedbackRecord(  # type: ignore
+                    rg.Record(  # type: ignore
                         fields={
                             self._id: instruction_id,
                             self._instruction: input["instruction"],
                             self._generation: generation,
                         },
-                    )
+                    ),
                 )
-        self._rg_dataset.add_records(records)  # type: ignore
+        self._dataset.log(records)  # type: ignore
```

Comment:
Should this not be `dataset.records.log` ?

Reply:
Yes, nice catch! It didn't fail tho, so most likely there's some missing exception around there